experimental/packages/opentelemetry-sdk-metrics-base/src/state/MeterProviderSharedState.ts

```diff
@@ -30,7 +30,15 @@ export class MeterProviderSharedState {
 
   metricCollectors: MetricCollector[] = [];
 
-  meters: Meter[] = [];
+  meterSharedStates: MeterSharedState[] = [];
 
   constructor(public resource: Resource) {}
+
+  getMeterSharedState(instrumentationLibrary: InstrumentationLibrary) {
+    // TODO: meter identity
```

Comment:
Do we have an issue for meter identity that can be referenced here so it isn't forgotten?

Reply:
I'll update the comment to include a reference to https://github.com/open-telemetry/opentelemetry-js/issues/2593.